src/screens/Game.jsx

```diff
@@ -1,19 +1,48 @@
-import React, { useEffect, useCallback, useState } from "react";
+import React, { useEffect, useCallback, useState, useContext } from "react";
 import { StyleSheet, View, SafeAreaView, Alert } from "react-native";
 
 import { getGame } from "../utils/api";
+import { listenGameEvent, closeConnection } from "../utils/mercure";
 
-import Title from "../components/Title";
-import Board from "../components/Board";
+import HeaderGame from "../components/HeaderGame.jsx";
+import Board from "../components/Board/Board.jsx";
+import {
+  GAME_FINISHED,
+  GAME_STARTED,
+  GAME_WAITING_OPPONENT,
+  NOT_YOUR_TURN,
+} from "../constants/game";
+import { PlayerContext } from "../providers/PlayerProvider";
 
 const GameScreen = ({ route }) => {
+  const [player] = useContext(PlayerContext);
   const [game, setGame] = useState();
 
+  const getStateStatusFromGame = (game) => {
+    // We have so many status & boolean from API.
```

Comment:
So, will it be possible to clarify the return of the API? (Not now!)

Reply:
I'm agree ,to do that I have to send `playerId` on `/games/:id` route and compute Response from this id.